src/stories/clickable-list/clickable-list.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { RawActiviteOption } from "interface/RawActiviteOption";
+import { getStoryFactory } from "stories/getStory";
+import { ClickableList } from "ui";
+import activites from "./activites.json";
+
+const options: RawActiviteOption[] = activites;
+
+const { meta, getStory } = getStoryFactory({
+    sectionName: "Composants",
+    "wrappedComponent": { ClickableList },
+});
+
+export default meta;
+
+export const Default = getStory({
+    options: options,
+    handleChange: () => console.log("handleChange"),
+    createActivity: () => console.log("createActivity"),
```

Comment:
console.log needed here ?

Reply:
error  Unexpected empty arrow function 'handleChange'    @typescript-eslint/no-empty-function
=> Just because the lint doesn't accept empty function and the story needs to have the attribute setted